packages/tables/resources/views/components/search-input.blade.php

```diff
@@ -15,6 +15,7 @@ class="pointer-events-none absolute inset-y-0 left-0 flex h-9 w-9 items-center j
             placeholder="{{ __('tables::table.fields.search_query.placeholder') }}"
             type="search"
             autocomplete="off"
+            dusk="filament-tables-search-input"
```

Comment:
Does this really need a dusk selector if the 2nd parent (`div`) has the `filament-tables-search-input` CSS class?

Reply:
I guess not then :)